README.md

```diff
@@ -29,12 +29,6 @@ This project is a full-featured form builder application built with Next.js. It
 
 ## Installation
 
-### Prerequisites
-
-- Node.js
-- PostgreSQL
-- Docker (optional, for containerization)
-
```

Comment:
Nodejs and PostgreSQL are required for the application to functions. In fact it would be better not only to mention the technologies but also the specific version as well.